.github/workflows/run-k8s-tests-on-zvsi.yaml

```diff
@@ -0,0 +1,72 @@
+name: CI | Run kubernetes tests on IBM Cloud Z virtual server instance (zVSI)
```

Comment:
Similar question to the one above about re-using the workflows we have, but adding a different value for the OS where it'd be running.